src/Aspire.Hosting.Azure/AzureProvisioningOptions.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Azure.Provisioning;
+
+namespace Aspire.Hosting.Azure;
+
+/// <summary>
+/// Options to configure the behavior of Azure resources.
+/// </summary>
+/// <remarks>
+/// These options can be configured using https://learn.microsoft.com/dotnet/core/extensions/options#use-di-services-to-configure-options
+/// to customize how Azure resources are provisioned.
+/// </remarks>
+public sealed class AzureProvisioningOptions
```

Comment:
Not sure this should be provisioning. 
```suggestion
public sealed class AzureResourceOptions
```

Reply:
Done.